### DISCUSSION TITLE
SEBE functionnalities and calculation of irradiation per m²

### DISCUSSION THREAD

**Discussion by apollineferry** [UMEP-dev/UMEP#576] [Q&A]
Hi, 
I am using SEBE in order to obtain irradiation map in mountainous areasand I have several questions about capacities/functionalities of SEBE. 

1- Until now I have calculated irradiation per m² on flat roofs. I only needed to calculate the total irradiation received by one roof and the total roof surface. I want now to obtain the irradiation per m² on inclined surfaces (roofs). To calculate the actual roof area per pixel, I need to consider the pixel surface and the slope of the surface : Actual_area = Area_pixel/ cos(Slope)
Then, to obtain the irradiation per m² of roof for roof pixel, I did :
Energy_flat_surface_per_m² = Energy_per_pixel/ pixel_surface
And to calculate the irradiation per m² of inclined roof for roof pixel, I thought to do :
Energy_inclined_surface_per_m² = Energy_per_pixel / Actual_area = Energy_per_pixel*cos(Slope)/ Area_pixel
However, in the tutorial, it is written that Energy_inclined_surface_per_m² = Energy_per_pixel/ (Area_pixel*cos(Slope))
Why irradiation on inclined surface is calculated in this way and what is wrong in the way I wanted to do  ?

2 – As I am focusing on mountainous area, I wanted to know if it’s possible with this tool to put a different albedos for different surfaces or if it’s only possible to put a mean albedo for all the surfaces ?

3 – I wanted to investigate the PV production in mountainous areas. I wanted to know if it’s possible to obtain hourly values of irradiation ? Or if the only way is to modify the weather file and select only one hour ?

4 – I saw that a publication was done on SEBEpv (https://doi.org/10.1016/j.solener.2020.01.075), that could calculate the PV production. Is this tool ready to use and is it possible to freely use it and have access to it ?

**Comment by biglimp** [comment]
1. Because the energy recieved by a inclined surface differ from that of a flat surface (no area considered)
2. No, at the moment only one albedo is possible
3. No, the model is not set up that way, unfortunately
4. We never had time to properly implement this to the tool but the code is actually there if you want to try out. In UMEP.py there are a couple of lines that needs to be uncommented (e.g. 52 and more). Look for #MRevesz or SEBEpv. You need to restart QGIS to make it work. LEt us know if ot is working and maybe we can activate this part of the tool as well.

**Comment by apollineferry** [comment]
Thank you for your answer. 

Concerning the first question, I have tested with different pixel resolution and it give similar value. I have then concluded that the value given is irradiation per square meter (kWh/m²) on surfaces and not irradiation per pixel (in kWh/pixel) (otherwise the value should change depending on the size of the pixel). 

I have not try yet to use SEBEpv, but I will try in the next week if I have time and let you know of the results.
